grid-editor/src/presenter/presenter.ts

```diff
@@ -24,29 +19,9 @@ export abstract class Presenter {
     public abstract erase(): void
 }
```

Comment:
Shouldn't there be an abstract present method?

Reply:
There's no single present method signature that covers all presenters. Because `PropertyPresenter`s needs a bounding box parameter to know where it should render itself, whereas `ShapePresenter` doesn't. Presenter is utility class that provide some share internal functionality.